data/saas/config/shipstation_config.yml

```diff
@@ -0,0 +1,75 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Shipstation
+  type: shipstation
+  description: A sample schema representing the Shipstation connector for Fides
+  version: 0.1.1
+
+  connector_params:
+    - name: api_key
+    - name: api_secret
+
+  external_references:
+    - name: customer_id
+      label: customerId Field
+      description: The dataset reference to the Shipstation customerId
+
+  client_config:
+    protocol: https
+    host: ssapi.shipstation.com
+    authentication:
+      strategy: basic
+      configuration:
+        username: <api_key>
+        password: <api_secret>
+
+  test_request:
+    method: GET
+    path: /customers
+
+  endpoints:
+    - name: orders
+      requests:
+        read:
+          method: GET
+          path: /orders
+          query_params:
+            - name: customerName
+              value: <customer_name>
+            - name: pageSize
+              value: "50"
+            - name: page
+              value: 1
+          data_path: orders
+          postprocessors:
+            - strategy: filter
+              configuration:
+                field: customerId
+                value:
+                  dataset_reference: <customer_id_value>
+          pagination:
+            strategy: offset
+            configuration:
+              incremental_param: page
+              increment_by: 1
+          param_values:
+            - name: customer_name
+              references:
+                - dataset: <instance_fides_key>
+                  field: customer.name
+                  direction: from
+            - name: customer_id_value
+              references:
+                - dataset: <instance_fides_key>
+                  field: customer.customerId
+                  direction: from
+        after: ["<instance_fides_key>.customers"]
```

Comment:
We don't need to specify and `after` for access requests. Fides will determine the correct order based on the `param_values` and relationships between the collections. The order of the endpoints also doesn't affect execution, but I like to put the endpoints in the order that they'll be visited. So in this case, define `customer` first, followed by `orders`